modules/administration-guide/pages/configuring-network-policies.adoc

```diff
@@ -44,6 +44,66 @@ spec:
 The default is `{prod-namespace}`.
 <2> The empty `podSelector` selects all Pods in the {orch-namespace}.
 
+* Optional: In case you applied link:https://docs.openshift.com/container-platform/{ocp4-ver}/networking/network_policy/multitenant-network-policy.html[Configuring multitenant isolation with network policy], you additionally must apply `allow-from-openshift-apiserver` and `allow-from-workspaces-namespaces` NetworkPolicies to `{prod-namespace}`. 
+The `allow-from-openshift-apiserver` NetworkPolicy allows incoming traffic from `openshift-apiserver` namespace to the `devworkspace-webhook-server` enabling webhooks.
+The `allow-from-workspaces-namespaces` NetworkPolicy allows incoming traffic from each user project to `che-gateway` pod.
++
+.`allow-from-openshift-apiserver.yaml`
+====
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-from-openshift-apiserver
+  namespace: {prod-namespace}   <1>
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: devworkspace-webhook-server   <2>
+  ingress:
+    - from:
+        - podSelector: {}
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: openshift-apiserver
+  policyTypes:
+    - Ingress
+----
+====
++
+<1> The {prod-short} namespace.
+The default is `{prod-namespace}`.
+<2> The `podSelector` only selects webhook-server pods
```

Comment:
That's a bit unclear, podSelector is empty but description mentions selection only webhook-server pods.

Reply:
Reason for my description is that line 63, where the reference of <2> is, is a child of the podSelector in line 61.

Technically `spec.podSelector` applies to / selects the destination pods. Line 66 on the other hand applies to pods that may start a network interaction.